src/controller/python/chip-device-ctrl.py

```diff
@@ -762,6 +762,40 @@ def resolve(fabric_id: int, node_id: int) -> Dict[str, Any]:
     except Exception as e:
         return __get_response_dict(status = StatusCodeEnum.FAILED, error = str(e))
 
+def zcl_command(cluster: str,command: str, node_id: int, endpoint_id: Optional[int] = 1, group_id: Optional[int] = 0, args: Optional[dict] = {}) -> Dict[str, Any] :
```

Comment:
Please refrain from using `args` as a variable name since python methods can have a have a built in *args component and doc strings contain args sections.

Reply:
Ack